### DISCUSSION TITLE
Reponun Private olup yapacakların sonradan eklenmesi

### DISCUSSION THREAD

**Discussion by Musa-Sina-Ertugrul** [karatasy/ulak-avionic#9] [General]
Eğer böyle yapmazsak fikirlerimiz çalınabilir

**Comment by karatasy** [comment]
yeni gelecek cok kisi var onlarin adaptasyonu icin acik tutalim

ama bu adimi biraz daha gelistirdikten sonra yapabiliriz

 ozellikle 3uncu sensor takildiktan sonra kesinlikle yapmaliyiz
